scripts/setup_R.sh

```diff
@@ -68,7 +69,9 @@ if [ ! -x "$(command -v r)" ]; then
 
     # Clean up
     # shellcheck disable=SC2086
-    apt-get remove --purge -y ${BUILDDEPS}
+    if [ "${PURGE_BUILDDEPS}" == "true" ]; then
```

Comment:
I think this should be opt-out, not opt-in.
@cboettig  Could you please update this to `if [ "${PURGE_BUILDDEPS}" != "false" ]; then`

Reply:
Done in #753